site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -1021,7 +1019,7 @@ private function deleteGradeable() {
         // this will cleanup the build files
         $this->enqueueBuildFile($g_id);
 
-        $this->core->redirect($this->core->buildNewUrl([$semester, $course]));
+        $this->core->redirect($this->core->buildNewUrl([]));
```

Comment:
Since the default parameter for `buildNewUrl` is an empty array, `buildNewUlr($parts = array(),...` you don't have pass in an empty array, you can just call the function without any params.

Reply:
Sure! A nice suggestion. Yes, I should do that. Will make the change.